bazel/_logging_threshold_test_main.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2024 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import subprocess
+import sys
+import tempfile
+
+_OK_TEST_REGEX = r"^-+.*Ran ([\d]+) tests* in ([\d.]+)s.*OK(?: \(skipped=(\d+)\))?\n$"
+
+_SKIP_TESTS = [
+    "_rpc_part_1_test",
```

Comment:
Skipping the `_rpc_part_1_test` _and_ the `_rpc_part_2_test` seems a bit excessive. These are the most basic unit tests we have. If we can't run ensure that logging regressions don't happen in those, I'm not sure how much value these tests have.

Reply:
I'll Actively work to get logging Issue with _rpc_part_1 and _rpc_part_2 resolved .
We can move rest, so that logging tests start running for rest of tests.

Reply:
What's the current issue with `_rpc_part_1` and `_rpc_part_2`?

Reply:
@XuanWang-Amos  We have some logs still in output . Need to work with core team on those.